packages/terra-button-group/package.json

```diff
@@ -27,13 +27,13 @@
     "react": "^15.4.2",
     "react-dom": "^15.4.2",
     "terra-base": "^2.10.0",
-    "terra-button": "^1.14.0"
+    "terra-button": "^x.0"
```

Comment:
I'm duplicating only the themeable styles but ButtonGroup still relies on the Button component because there is some logic in Button that I don't think we should duplicate.